Assignment 1/Exercise3.java

```diff
@@ -0,0 +1,34 @@
+import java.io.Console;
+
+/**
+ * Exercise3
+ */
+public class Exercise3 {
+
+    private static int start;
```

Comment:
Great!, We never talked about it but its better to have the variables only in the scope they are required, so this will be an optional change for this time but required after our class 5. Other than that awesome!.